src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -870,7 +900,32 @@ public static implicit operator (object a, object b)(NewStruct value)
         return new NewStruct(value.a, value.b);
     }
 }";
-            await TestInRegularAndScriptAsync(text, expected, options: this.PreferImplicitTypeWithInfo(), testHost: host);
+            var test = new VerifyCS.Test
+            {
+                TestCode = text,
+                FixedCode = expected,
+                TestHost = host,
+                ExpectedDiagnostics =
+                {
+    // /0/Test0.cs(6,22): error CS0103: The name 'Goo' does not exist in the current context
+    DiagnosticResult.CompilerError("CS0103").WithSpan(7, 22, 7, 25).WithArguments("Goo"),
+    // /0/Test0.cs(6,32): error CS0103: The name 'Bar' does not exist in the current context
+    DiagnosticResult.CompilerError("CS0103").WithSpan(7, 32, 7, 35).WithArguments("Bar"),
+                },
```

Comment:
I'd do one of the following:

- Get rid of the diagnostics from the test.
- Use the `{|CS0103:code|}` syntax.
- Use `{|#0:code|}` syntax and use `WithLocation(0)` instead of `WithSpan`.

Will let you and @sharwell decide the appropriate change if any.

Reply:
> Get rid of the diagnostics from the test.

in this case, the tewst is intentionally testint out what we do with incorrect code.  so i want to keep that :)

> Use the {|CS0103:code|} syntax.

I'm fine with that.  but i'm also fine with how this looks (which is also how the test harness outputs the diags).  i'm honestly not wanting to spend much effort here.

Reply:
We use explicit `ExpectedDiagnostics` for compiler errors that are likely the result of an incorrectly written test, and `{|CS0103:...|}` for cases where the error is intentionally part of the test input/output.

Reply:
ah ok.  will do.